paasta_tools/marathon_tools.py

```diff
@@ -470,7 +472,8 @@ def load_service_namespace_config(service, namespace, soa_dir=DEFAULT_SOA_DIR):
         'mode',
         'discover',
         'advertise',
-        'extra_healthcheck_headers'
+        'extra_healthcheck_headers',
+        'balance'
```

Comment:
Are you saying there should be one or there is one? Because it looks right to me (at least based on what it was before)